AGV:
  radius: 80
  wheelbase: 588
PLC:
  ams_id: 192.168.100.100.1.1
  fps: 30
  hostname: 192.168.100.191
  ip: 192.168.100.100
  password: '1'
  route_name: RouteToMyPC
  sender_ams: 1.2.3.4.1.1
  username: Administrator
